draft-ietf-masque-h3-datagram.md

```diff
@@ -731,7 +835,7 @@ DATAGRAM                       -------->
 STREAM(44): CAPSULE             -------->
   Capsule Type = REGISTER_DATAGRAM_CONTEXT
   Context ID = 2
-  Extension String = "ip=192.0.2.42,port=443"
+  Context Extension = {IP_COMPRESSION=192.0.2.42:443}
```

Comment:
Longer term, we might want to explain the presentation format for extensions but for now I think this is ok.

Reply:
Agreed, these example are likely to be removed before publication.